### ISSUE TITLE
Linux Platforms will always return errors despite libvlc and libvlccore being properly installed on the system

### ISSUE THREAD

**Issue by BeastlyGhost** [polybiusproxy/hxCodec#130]
linux platforms will always return 
![image](https://user-images.githubusercontent.com/45212377/171174240-3d3a1ece-f734-4ef4-b55f-793b7dfecc4f.png)
this when trying to compile for them, forcing you to simply leave video support on windows-only platforms

**Comment by realAliDevex** [comment]
> I fixed it by replacing some of the librarys with working files and by installing `vlc-core` and `vlc` package on Fedora 36!

I also made 2 pull requests to fix it. :)

**Comment by BeastlyGhost** [comment]
> > I fixed it by replacing some of the librarys with working files and by installing `vlc-core` and `vlc` package on Fedora 36!
> 
> I also made 2 pull requests to fix it. :)

![image](https://user-images.githubusercontent.com/45212377/171311651-184728af-d8d0-4434-9059-845b7cf857aa.png)
they work, but when trying to play the video, they simply crash

**Comment by BeastlyGhost** [comment]
really hoping it gets merged though, this is something!

**Comment by realAliDevex** [comment]
> really hoping it gets merged though, this is something!

Yeah, hope that it gets accepted! I used the original (working) libvlc* files because they should work. :)

**Comment by realAliDevex** [comment]
> really hoping it gets merged though, this is something!

Finally! It got accepted! :)
Thanks @polybiusproxy. <3

**Comment by Gibberditoslumber** [comment]
> linux platforms will always return ![image](https://user-images.githubusercontent.com/45212377/171174240-3d3a1ece-f734-4ef4-b55f-793b7dfecc4f.png) this when trying to compile for them, forcing you to simply leave video support on windows-only platforms

Use ld To Make An Actual Linker Script

**Comment by BeastlyGhost** [comment]
already fixed

**Comment by Gibberditoslumber** [comment]
oh okay
